src/Backend/PostponeRuntimeBackend.php

```diff
@@ -79,7 +79,7 @@ public function onEvent(Event $event = null)
     {
         reset($this->messages);
 
-        while (list($key, $message) = each($this->messages)) {
+        foreach ($this->messages as $key => $message) {
             $this->handle($message, $this->dispatcher);
 
             unset($this->messages[$key]);
```

Comment:
You cant unset like this in a foreach, you should move this line outside the foreach in the form:

```
$this->messages = [];
```

Reply:
> You cant unset like this in a foreach

why not? oO

Reply:
Because you are removing elements on the foreach and php no longer knows what element is next.

On the old code it was doing a foreach until there are no elements on the array. That's why it was working before

Reply:
Other option would be doing a while with "there are elements on the array" condition, but IMO a foreach is cleaner

Reply:
still failing :(